client/guidestours/style.scss

```diff
@@ -58,3 +84,47 @@ $zoom-scale: 5; // the multiplier determining the size of the animated rings
 .guidestours__pointer__ring:after {
 	animation-delay: #{ $animation-speed / 4 };
 }
+
+.guidestours__buttonrow__choice {
+	button {
+		width: 48%;
+	}
+	button:nth-child(1) {
+		margin-right: 4%;
+	}
+	button.guidestours__button__secondary {
+		background: $gray-light;
```

Comment:
😄 We had added a `secondary` button to the `Button` component already in this PR but revised that so it would be a more local change. Would be interested to hear what others think whether a) having a gray button here is defensible and b) that should go into the component.